generator-prisma-client/aws-lambda-sst-esbuild/README.md

```
@@ -0,0 +1,153 @@
+Run
+
+```
+pnpm run build
+```
+
+### 6. Run with SST
+
+- Ensure you set up your AWS Credentials and have the AWS CLI installed and logged in (see [sst docs](https://sst.dev/docs/aws-accounts))
+- Set up secrets in your SST project. You can do this by:
+  - Manually adding the `DIRECT_URL` connection string that you previously set in your `.env` file, via:
+  ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)